src/Storage/Device/Local.php

```diff
@@ -410,4 +410,29 @@ public function getPartitionTotalSpace(): float
     {
         return \disk_total_space($this->getRoot());
     }
+
+    /**
+     * Get all files inside a directory.
+     *
+     * @param  string  $dir Directory to scan
+     * @return string[]
+     */
+    private function getFiles(string $dir): array
+    {
+        if (! (\str_ends_with($dir, DIRECTORY_SEPARATOR))) {
+            $dir .= DIRECTORY_SEPARATOR;
+        }
```

Comment:
Didn't you mention ending in `DIRECTORY_SEPARATOR` was optional for `scandir` ?

Reply:
It is optional for scandir. I added it so later I can do `$absolutePath = $dir . $file;` without worrying if `/` is there or not:

https://github.com/utopia-php/storage/blob/c8afbaf2dadeae50e863edf332412e266dd63b87/src/Storage/Device/Local.php#L433

Reply:
All good. Can you remove the comments from test so that I can merge it?